docs/basic-usage.md

```diff
@@ -76,6 +76,29 @@ cd pre-existing-project
 poetry init
 ```
 
+### Operating modes
+
+Poetry can be operated in two different modes. The default mode is the **package mode**, which is the right mode
+if you want to package your project into an sdist or a wheel and perhaps publish it to a package index.
+In this mode, some metadata such as `name` and `version`, which are required for packaging, are mandatory.
+Further, the project itself will be installed in editable mode when running `poetry install`.
+
+If you want to use Poetry only for dependency management but not for packaging, you can use the **non-package mode**:
+
+```toml
+[tool.poetry]
+package-mode = false
+```
```

Comment:
Will the setting work in `poetry.toml` as well? If so, I would add an example for  that as well.

Reply:
I don't think so. Actually, I don't think it would be a good idea to put it in `poetry.toml` because it's clearly a project setting and not a user setting. (It's not like user A wants to have this setting for project X while user B does not want this setting for project X.)

Reply:
I meant `poetry.toml` inside a project directory. I did not test it yet, but I assume settings in `poetry.toml` are loaded the same as ones in `pyproject.toml`'s `tool.poetry` section.